NETWORK/NetworkAddMapObjectToSynchronisedScene.md

```diff
@@ -0,0 +1,45 @@
+---
+ns: NETWORK
+aliases: ["0x45F35C0EDC33B03B"]
+---
+## _NETWORK_ADD_MAP_OBJECT_TO_SYNCHRONISED_SCENE
+
+```c
+// 0x45F35C0EDC33B03B
+void _NETWORK_ADD_MAP_OBJECT_TO_SYNCHRONISED_SCENE(int netScene, Hash modelHash, float x, float y, float z, cs_type(float) char* animDict, char* animName, float p7, float p8, int flags);
```

Comment:
Looking at the parameter ordering, which is similar to other methods such as [NETWORK_ADD_PED_TO_SYNCHRONISED_SCENE](https://github.com/citizenfx/natives/blob/master/NETWORK/NetworkAddPedToSynchronisedScene.md) and [TASK_PLAY_ANIM](https://github.com/citizenfx/natives/blob/master/TASK/TaskPlayAnim.md) `p7` may be `blendInSpeed` and `p8` may be `blendOutSpeed`.

The decompiled method structures (where these parameters get applied) for both `NETWORK_ADD_PED_TO_SYNCHRONISED_SCENE` and this are also similar.

Reply:
thank you, i've added them